bin/_mkasset.bat

```diff
@@ -25,30 +25,31 @@ title %MINDBENDER_PROJECTPATH% / %MINDBENDER_ASSET%
 cls
 
 echo+
-echo   %MINDBENDER_ASSET% -----------
+echo    %MINDBENDER_ASSET% -----------
 echo+
 echo+
-echo   1. Type application and task.
-echo   2. Press [Enter] to launch application.
+echo    1. Type application and task.
```

Comment:
I think I understand why there are all these small changes on spaces/tabs, but for posterity could you make a note about why you made these?